packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -518,6 +518,7 @@
 	"executionDetails.confirmMessage.headline": "Delete Execution?",
 	"executionDetails.confirmMessage.message": "Are you sure that you want to delete the current execution?",
 	"executionDetails.deleteExecution": "Delete this execution",
+	"executionDetails.enqueuedMessage": "Execution hasn't started yet.",
```

Comment:
As a user, I'd like to know why the execution hasn't started - if there's a problem or if this is behaving as intended, especially if I'm unaware of the concurrency capacity and I might be seeing this message for a while. `Execution was rate-limited`? Might need product input.